src/EventHub/EventHub/Cmdlets/Clusters/NewAzureEventHubCluster.cs

```diff
@@ -56,46 +69,66 @@ public class NewAzureRmEventHubCluster : AzureEventHubsCmdletBase
 
         public override void ExecuteCmdlet()
         {
-
-            PSEventHubClusterAttributes cluster = new PSEventHubClusterAttributes(); 
-
-            if (ParameterSetName.Equals(ClusterPropertiesParameterSet))
+            if(ParameterSetName.Equals(ClusterPropertiesParameterSet) && ResourceId != null)
             {
-                cluster.Location = Location;
-                if (this.IsParameterBound(c => c.Capacity))
-                {
-                    cluster.Sku.Capacity = Capacity;
-                }
-                else
-                {
-                    cluster.Sku.Capacity = 1;
-                }
-
-                if (this.IsParameterBound(c => c.Tag))
-                {
-                    cluster.Tags = TagsConversionHelper.CreateTagDictionary(Tag, validate: true);
-                }                               
+                throw new PSNotImplementedException("-ResourceId parameter would be deprecated for New-AzEventHubCluster");
```

Comment:
This line makes ResourceId can't be used by user in current spring. Please revert it.

Reply:
Okay, I will remove this line of code.